src/App.js

```diff
@@ -4,6 +4,16 @@ import Questions from './components/polls/questions/Questions';
 import WelcomePage from './components/Welcome Page/WelcomePage';
 import Storybook, { CarouselItem } from './components/storybook/Storybook';
 import Emotions from "./components/emotions/Emotions";
+
+import OnlineUsers from "./components/onlineUsers/OnlineUser";
```

Comment:
I would’ve called it UserPresence since they’re not always “online”. But that’s a minor detail. No actions required.